ArchiSteamFarm/Bot.cs

```diff
@@ -2836,6 +2840,290 @@ public sealed class Bot : IAsyncDisposable {
 			}
 		}
 
+		private void OnInventoryChanged() {
+			Utilities.InBackground(CardsFarmer.OnNewItemsNotification);
+
+			if (BotConfig.BotBehaviour.HasFlag(BotConfig.EBotBehaviour.DismissInventoryNotifications)) {
+				Utilities.InBackground(ArchiWebHandler.MarkInventory);
+			}
+
+			if (BotConfig.CompleteTypesToSend.Count > 0) {
+				Utilities.InBackground(SendCompletedSets);
+			}
+		}
+
+		private async Task<HashSet<uint>?> GetPossiblyCompletedBadgeAppIDs() {
```

Comment:
Consider making this method public to use in plugins.

Reply:
I'm not sure, why you would possibly need this one. Can you elaborate?

Reply:
For the same thing why you use it here - to optimize by not loading all the appIDs, but only those for each we can possibly have completed sets.